stdlib/typing.pyi

```diff
@@ -24,10 +23,15 @@ from types import (
 )
 from typing_extensions import Never as _Never, ParamSpec as _ParamSpec
 
-if sys.version_info >= (3, 10):
-    from types import UnionType
 if sys.version_info >= (3, 9):
     from types import GenericAlias
+if sys.version_info >= (3, 10):
+    from types import UnionType
+if sys.version_info >= (3, 13):
+    # NOTE: Starting with Python 3.13 we can just re-export these from contextlib
+    from contextlib import AbstractAsyncContextManager as AsyncContextManager, AbstractContextManager as ContextManager
+else:
+    from contextlib import AbstractAsyncContextManager, AbstractContextManager
```

Comment:
@JelleZijlstra Not sure I'm a fan of this `else`, maybe it's cleaner to move the import down to where the derived protocols are defined or keep the whole `if`/`else` down there despite the import missing from the top of the file this way. The protocol definitions have to pretty much stay where they are, I tried to move them further up before.

Reply:
Yes, I'd prefer to move the imports down to where the protocols are.